### PR TITLE
Set up NDT web deployments

### PR DESCRIPTION
**Pull request by mythmon**
This sets up CircleCI to deploy the website on tags, and makes a new tagged release that we can use to trigger the deploy once this merges.

### CODE REVIEW

**Review by rehandalal — APPROVED** [review]
bors r+

File: .circleci/config.yml [hunk 1]
```diff
@@ -140,6 +131,47 @@ jobs:
...
+        type: env_var_name
+      bucket-name:
+        type: env_var_name
+      distribution-id:
+        type: env_var_name
+      secret-access-key:
+        type: env_var_name
+
+    docker:
+      - image: circleci/python:3
```
Comment (sciurus) [inline .circleci/config.yml]: There's an image with the aws cli already you could use if you like.

https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-docker.html#cliv2-docker-install

Comment (mythmon) [reply .circleci/config.yml]: thanks!

**Comment by mythmon** [comment]
bors r= @rehandalal

File: .circleci/config.yml (hunk 1 above)
Comment (mythmon) [reply .circleci/config.yml]: Turns out that amazon/aws-cli is a bit too minimal for our use cases. I got this error:

> tar utility is not present in this image but it is required. Please install it to have workflow workspace capability.

https://app.circleci.com/pipelines/github/mozilla-extensions/normandy-devtools/2487/workflows/298f5515-1685-4e8a-82ca-2c15a96e927f/jobs/7402

**Pull request merged**